src/partner-access/partner-access.model.ts

```diff
@@ -0,0 +1,21 @@
+import { IPartner } from 'src/partner/partner.model';
+import { IUser } from 'src/user/user.model';
+
+export interface IPartnerAccess {
+  id: string;
+  createdBy: CreatedBy;
+  partner: IPartner;
+  user?: IUser;
+  activatedAt?: Date | string;
+  accessCode: string;
+  featureLiveChat: boolean;
+  featureTherapy: boolean;
+  therapySessionsRemaining: number;
+  therapySessionsRedeemed: number;
+}
+
+interface CreatedBy {
```

Comment:
why is CreatedBy it's own interface? I thought this was just a relationship to the PartnerAdmin model, so the interface would be the PartnerAdmin one. might be missing something though?